mne/viz/tests/test_3d.py

```diff
@@ -135,8 +135,8 @@ def test_plot_sparse_source_estimates(renderer):
     stc_data[1, 4] = 2.
     vertices = [vertices[inds], np.empty(0, dtype=np.int)]
     stc = SourceEstimate(stc_data, vertices, 1, 1)
-    surf = plot_sparse_source_estimates(sample_src, stc, bgcolor=(1, 1, 1),
-                                        opacity=0.5, high_resolution=False)
+    surf, _ = plot_sparse_source_estimates(sample_src, stc, bgcolor=(1, 1, 1),
```

Comment:
This is a backward incompatible change to a public function. Usually we do a `return_whatever=False` kwarg to allow this.

Or, if you don't ever actually need to give the second output to users, you can do the following to avoid needing any new `kwarg`:
```
def plot_sparse_source_estimates(...):
    """...
    """
    return _plot_sparse_source_estimates(...)[0]

def _plot_sparse_source_estimates(...):
    ... # old code of plot_sparse_source_estimates
    return surf, pd
```